src/PoseNet/index.js

```diff
@@ -76,9 +76,9 @@ class PoseNet extends EventEmitter {
       }
       if (this.detectionType === 'single') {
         this.singlePose();
+      } else {
```

Comment:
I would refactor all this promise:

Also avoiding else in the code is so good for readibility.

https://medium.com/@jamousgeorges/dont-use-else-66ee163deac3

```JS 
async load() {
   this.net = await.posenet.load(this.multiplier);
   if (!this.video) {
         return this;
   }

   if (this.video.readyState === 0) {
    this.vide.onloadeddata = () => Promise.resolve();
   }

   if(this.detectionType === 'single') {
       this.singlePose();
    }
    else {
       this.multiPose();
   }

   return this;
}
```